docs/specs/servers/blockchain/blockchain-server-api.md

```diff
@@ -276,3 +276,135 @@ The POST request body should be in JSON format with the following structure:
 
 * `400 Bad Request` - some parameters in request body were missed or wrong.
 * `401 Unauthorized` - projectID verification error.
+
+## Conversion
+
+Endpoints to make convert token transactions.
+
+### List of tokens available for conversion
+
+`GET /v1/convert/tokens`
+
+#### Path parameters
+
+* `projectId` - is the project identifier
+* `chainId` - filter by chain. CAIP-10 format.
+
+#### Success response body:
+
+* `tokens` - list of objects which define available tokens:
+    * `name` - token name.
+    * `symbol` - token symbol.
+    * `address` - contract address of token.
+    * `chainId` - ID of chain in CAIP-10 format.
+    * `decimals` - decimals for amount supported by a given token.
+    * `logoURI` - URL of the token icon.
+    * `eip2612` - (Optional for ERC-20 tokens) value is `true` if the token supports [eip-2612](https://eips.ethereum.org/EIPS/eip-2612)
+
+#### Response error codes:
+
+* `400 Bad Request` - some parameters in request body were missed or wrong.
+* `401 Unauthorized` - project ID verification error.
+
+### Requesting conversion quotes
+
+`GET /v1/convert/quotes`
```

Comment:
@geekbrother Can we have more info about the inpoint under their titles? We have currently three convert-related endpoints and tbh it's not obvious to me which one to use for which action. Can we pair about it and make sure that their spec is handling all the states that we need.

- `GET /v1/convert/quotes`
- `POST /v1/convert/build-approve`
- `POST /v1/convert/build-transaction`

Reply:
Makes sense to update the description and add the swap flow on how they should be called.